docs/apm/custom-links.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[custom-links]]
+=== Custom links
+
+Custom links allow you to easily create dynamic links based on your specific APM data.
+Links can be filtered to only appear for specific services,
+and there's no limit to the number of links you can create.
```

Comment:
@formgeist didn't we agree to show 500 links?

Reply:
Indeed, we should clarify this in the copy.